<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>海天鹰播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="filelist.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;background:black;</string>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::NoDrag</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_live">
     <property name="title">
      <string>直播</string>
     </property>
     <addaction name="action_liveList"/>
     <addaction name="action_liveImport"/>
    </widget>
    <addaction name="action_open"/>
    <addaction name="action_openURL"/>
    <addaction name="menu_live"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_changelog"/>
    <addaction name="action_aboutQtAV"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_video">
    <property name="title">
     <string>视频</string>
    </property>
    <widget class="QMenu" name="menu_scale">
     <property name="title">
      <string>缩放</string>
     </property>
     <addaction name="action_scale0_5"/>
     <addaction name="action_scale1"/>
     <addaction name="action_scale1_5"/>
     <addaction name="action_scale2"/>
    </widget>
    <addaction name="action_fullscreen"/>
    <addaction name="action_info"/>
    <addaction name="action_capture"/>
    <addaction name="action_captureDirectory"/>
    <addaction name="action_capture16"/>
    <addaction name="menu_scale"/>
    <addaction name="action_rotateCW"/>
    <addaction name="action_rotateCCW"/>
   </widget>
   <widget class="QMenu" name="menu_sound">
    <property name="title">
     <string>音频</string>
    </property>
    <addaction name="action_volumeUp"/>
    <addaction name="action_volumeDown"/>
    <addaction name="action_volumeMute"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_video"/>
   <addaction name="menu_sound"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_changelog">
   <property name="text">
    <string>更新历史</string>
   </property>
  </action>
  <action name="action_aboutQtAV">
   <property name="text">
    <string>关于QtAV</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_volumeUp">
   <property name="text">
    <string>增加音量</string>
   </property>
  </action>
  <action name="action_volumeDown">
   <property name="text">
    <string>减小音量</string>
   </property>
  </action>
  <action name="action_volumeMute">
   <property name="text">
    <string>静音</string>
   </property>
  </action>
  <action name="action_fullscreen">
   <property name="text">
    <string>全屏</string>
   </property>
  </action>
  <action name="action_openURL">
   <property name="text">
    <string>打开URL</string>
   </property>
  </action>
  <action name="action_info">
   <property name="text">
    <string>信息</string>
   </property>
  </action>
  <action name="action_capture">
   <property name="text">
    <string>截图</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_rotateCW">
   <property name="text">
    <string>顺时针旋转</string>
   </property>
  </action>
  <action name="action_rotateCCW">
   <property name="text">
    <string>逆时针旋转</string>
   </property>
  </action>
  <action name="action_scale0_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.5 X</string>
   </property>
  </action>
  <action name="action_scale1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 X</string>
   </property>
  </action>
  <action name="action_scale2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 X</string>
   </property>
  </action>
  <action name="action_scale1_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.5 X</string>
   </property>
  </action>
  <action name="action_liveList">
   <property name="text">
    <string>列表</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="action_liveImport">
   <property name="text">
    <string>导入</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="action_capture16">
   <property name="text">
    <string>剧情连拍</string>
   </property>
  </action>
  <action name="action_captureDirectory">
   <property name="text">
    <string>截图目录</string>
   </property>
  </action>
  <action name="action_soundTrack">
   <property name="text">
    <string>音轨</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="filelist.qrc"/>
 </resources>
 <connections/>
</ui>
